src/components/RunGraph.vue

```diff
@@ -6,14 +6,24 @@
   import { onBeforeUnmount, onMounted, ref } from 'vue'
   import { RunGraphConfig } from '@/models/RunGraph'
   import { start, stop } from '@/objects'
+  import { RunGraphDomain } from '@/objects/domain'
+  import { emitter } from '@/objects/events'
   // import { WorkerMessage, worker } from '@/workers/runGraph'
 
   const props = defineProps<{
     config: RunGraphConfig,
+    domain?: RunGraphDomain,
+  }>()
+
+  const emit = defineEmits<{
+    (event: 'update:domain', value: RunGraphDomain): void,
   }>()
 
   const stage = ref<HTMLDivElement>()
 
+  emitter.on('domainCreated', domain => emit('update:domain', domain))
```

Comment:
Surprised you can't listen to a list of events:
```ts
  emitter.on(['domainCreated', 'domainUpdated'], domain => emit('update:domain', domain))
```

Reply:
I also felt like that should work. But mitt is purposefully tiny and doesn't have stuff like this. 

If we end up using event busses more I'll probably make our own that implements some of these nice devx features.